docs/v2/integrations/advanced-custom-fields.md

```diff
@@ -77,8 +77,8 @@ This is where we’ll start in PHP.
 ```php
 $post = Timber::get_post();
 
-if (isset($post->hero_image) && strlen($post->hero_image)) {
-    $post->hero_image = Timber::get_image($post->hero_image);
+if (isset($post->meta('hero_image')) && strlen($post->meta('hero_image'))) {
```

Comment:
I wonder whether we even need to set `$post->hero_image`. It’s not necessary, right? Because later, we call `post.meta('hero_image')` anyway. We could just delete the whole if call, which would make the whole example easier to understand. Instead, we could place the if call in Twig:

```twig
{% if post.meta('hero_image') %}
	<img src="{{ post.meta('hero_image').src | resize(500, 300) }}" />
{% endif %}
```

In addition to that, setting object properties dynamically isn’t something we should promote. If you wanted to use `post.hero_image` directly, the recommended way to do this would be to create a custom class that extends `Timber\Post` and then define a `hero_image` property or a `hero_image()` method there.

Reply:
I think that would make a lot of sense. I just "blindly" updated the example to use `meta` instead of relying on the magic call, but didn't really think about it

Reply:
@xavivars Good :). Do you want to update the PR or do you want me to do it?

Reply:
Please, go ahead. I completely forgot about this 😅